scripts/build_utils.sh

```diff
@@ -489,6 +489,13 @@ echo "deb [arch=amd64] http://us.archive.ubuntu.com/ubuntu/ $DIST-backports main
 echo "deb [arch=amd64] http://security.ubuntu.com/ubuntu $DIST-security main restricted universe multiverse" >> /etc/apt/sources.list
 env DEBIAN_FRONTEND=noninteractive apt-get update -y -o Acquire::Languages=none -o Acquire::Translation=none || true
 env DEBIAN_FRONTEND=noninteractive apt-get install -y gnupg
+
+set +e
+env DEBIAN_FRONTEND=noninteractive apt-get install -y curl
+curl -L https://dist.apache.org/repos/dist/dev/arrow/KEYS | apt-key add -
+apt-get install -y software-properties-common
```

Comment:
could you please add `curl` and `software-properties-common` to `EXTRAPACKAGES`? see https://github.com/ceph/ceph-build/blob/297f54a86ed061fafdfebcefdc919980c34c848f/scripts/build_utils.sh#L604

Reply:
:+1: